README.md

```diff
@@ -11,13 +11,14 @@ Additionally, it ***must*** have a `#description` method that returns a string.
 
 Additionally it ***should*** implement `#manifest_url` that shows where the manifest can be found. 
 
+Additionally it ***may*** implement `#manifest_metadata` to provide an array containing hashes of metadata Label/Value pairs. 
```

Comment:
Should this be ***should*** instead of ***may***?  The Presentation API says:
```
A manifest should have one or more metadata pairs associated with it describing the object or work.
```

Reply:
Yes, good catch. I've aligned it with IIIF.